packages/playground/src/components.d.ts

```diff
@@ -0,0 +1,65 @@
+/* tslint:disable */
+/**
+ * This is an autogenerated file created by the Stencil compiler.
```

Comment:
If it is auto generated do we need to commit it? Shouldn't it be part of some kind of build process